README.md

```diff
@@ -43,6 +43,22 @@ already have these, if not please download and run vcredist.x64.exe from
 [Microsoft](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads) to install them prior to running
 cbmc.
 
+### Linux
+
+For different linux environments, you have these choices:
+
+1. Install CBMC through the distribution's repositories, with the downside
+   that this might install an older version of cbmc, depending on what the
+   package maintenance policy of the distribution is, or
+2. Install CBMC through the `.deb` package build by each release, available
+   on the [releases](https://github.com/diffblue/cbmc/releases) page. To
+   do that, download the `.deb` package and run `apt install cbmc-x.y.deb`
+   with `root` privileges, with `x.y` being substituted for the version
+   you are attempting to install.
+   *NOTE*: This is only available on distributions that are using debian's
+   packaging tools.
```

Comment:
:-1: The debian packages are distro specific (due to link constraints against libc/libc++ and package dependency names), this needs to be noted here. We have two separate packages for Ubuntu 18.04 and Ubuntu 20.04, they shouldn’t be used anywhere else.

Reply:
Okay... I was unaware of any link constraints we have, can you expand on this? At least `libc` ABI should be stable, and I would go on a limb and say that `libc++` should be stable enough for our needs as well - why is it not?

But noted in any case, I'll update the documentation.

Reply:
@NlightNFotis libc/libc++ ABI is stable, but not backwards compatible. You can link binaries linked against libc against _newer_ versions of libc, but not older ones.